parse-speeches.rb

```diff
@@ -79,20 +82,47 @@ def parse_date(text)
 
 progress = ProgressBar.new("parse-speeches", ((to_date - from_date + 1) * 2).to_i)
 
+def parse_with_retry(interactive, parse, date, path, house)
+  begin
+    parse.call date, path, house
+  rescue Exception => e
+    puts "ERROR While processing #{house} #{date}:"
+    if not interactive
```

Comment:
Changing this to `raise unless interactive` makes this a neater 1 line, [favours `unless`](https://github.com/bbatsov/ruby-style-guide#unless-for-negatives), and [doesn't use `not`](https://github.com/bbatsov/ruby-style-guide#bang-not-not) :sparkles:

Reply:
I'm sure you can see my Python coming through... :)

Reply:
Just a bit :smiley: